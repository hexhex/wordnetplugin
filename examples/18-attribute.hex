%% this returns all words, of which a given word
%% is an attribute
%% POS: adjectives

%% the search number for attribute
%% is 18.

word("loud").
attribute(W,A) :- &wordnet[18,W](_,_,A,_), word(W).

attribute("loud","intensity"), 
attribute("loud","volume")?
