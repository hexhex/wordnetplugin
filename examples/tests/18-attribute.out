{attribute("loud","intensity"), attribute("loud","loudness"), attribute("loud","volume"), word("loud")}
